testing/test-config.js

```diff
@@ -58,12 +58,11 @@ export class TestConfig {
     this.runOnFirefox = this.platform.isFirefox.bind(this.platform);
     this.runOnSafari = this.platform.isSafari.bind(this.platform);
     this.runOnIos = this.platform.isIos.bind(this.platform);
-    this.runOnIe = this.platform.isIe.bind(this.platform);
 
     /**
      * By default, IE is skipped. Individual tests may opt in.
      */
-    this.skip(this.runOnIe);
```

Comment:
@rsimha Changes here might be relevant for your review, many of these are designed to be able to successfully merge this PR but retain the existing testing functionality.

Know you are also working on a PR to remove this in the future.

Reply:
Thanks for the heads up. I can confirm that this line has had the desired effect, and is resulting in [zero IE tests](https://github.com/ampproject/amphtml/pull/35317/checks?check_run_id=3231831293#step:4:801). (Once this PR lands, I can clean up the rest of the code via #35341.)

![image](https://user-images.githubusercontent.com/26553114/128070589-e76183e1-96ed-4a87-a24f-62f2219190ea.png)